playtime/init.lua

```diff
@@ -0,0 +1,88 @@
+playtime = {}
+
+local os = os
+local math = math
+local string = string
+
+local current = {}
+
+local key = "playtime:time"
+
+local S = minetest.get_translator("playtime")
+local C = minetest.colorize
+
+--get playtime
```

Comment:
Superfluous comment. Maybe "current" should be replaced with "session" in naming to make this clear.